okapi-common/src/main/java/org/folio/okapi/common/CqlUtil.java

```diff
@@ -80,19 +82,22 @@ public static CQLNode reducer(CQLNode vn1, CQLTermNode tn, Comparator<CQLTermNod
         return new CQLPrefixNode(prefix.getName(), prefix.getIdentifier(), n2);
       }
     } else {
-      throw new IllegalArgumentException("unknown type for CQLNode: " + vn1.toString());
+      throw new IllegalArgumentException("unknown type for CQLNode: "
+        + vn1.toString());
     }
   }
 
-  private static CQLNode reduceBoolean(CQLBooleanNode n1, CQLTermNode tn, Comparator<CQLTermNode> cmp) {
+  private static CQLNode reduceBoolean(CQLBooleanNode n1, CQLTermNode tn,
+                                       Comparator<CQLTermNode> cmp) {
+
     CQLNode n2 = null;
     CQLNode left = reducer(n1.getLeftOperand(), tn, cmp);
     CQLNode right = reducer(n1.getRightOperand(), tn, cmp);
 
-    ModifierSet mSet = new ModifierSet(n1.getOperator().toString().toLowerCase());
+    ModifierSet mset = new ModifierSet(n1.getOperator().toString().toLowerCase());
```

Comment:
Set is a word and must start with a capital letter, see above.

Reply:
Sanme here. Pedantic